base_tier_validation_correction/models/tier_validation.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+from odoo.tools.safe_eval import safe_eval
+
+
+class TierValidation(models.AbstractModel):
+    _inherit = "tier.validation"
+
+    def view_tier_correction(self):
+        self.ensure_one()
+        # Caller document
+        active_model = self.env.context.get("active_model")
+        active_id = self.env.context.get("active_id")
+        res_model = self.env["ir.model"].search([("model", "=", active_model)])
+        doc = self.env[active_model].browse(active_id)
+        # Prepare to redirect to Tier Correction
+        action = self.env.ref("base_tier_validation_correction.tier_correction_action")
```

Comment:
```suggestion
        action = self.sudo().env.ref("base_tier_validation_correction.tier_correction_action")
```
Add sudo() for fix right to access Action Window
![Selection_401](https://user-images.githubusercontent.com/24691983/139943298-80514cfa-0559-4a36-a4e2-3ef4ceda40b2.png)